bibliogra.py

```diff
@@ -219,9 +219,9 @@ def format_authors(persons, hilight):
 
     for person in persons[author_type]:
         authors_list.append(
-            " ".join(person.first() + person.middle() + person.last()))
+            " ".join(person.first_names + person.middle_names + person.last_names))
 
-    authors_list = [authors.decode("latex") for authors in authors_list]
```

Comment:
Why are we removing the decode call? Without it, the code returns "D\'iaz" instead of "Díaz".

Reply:
I removed it due to `AttributeError: 'str' object has no attribute 'decode'`, I didn't check name by name the results. Sorry.